ckanext/ckanext-apicatalog_scheming/ckanext/apicatalog_scheming/schemas/organization.json

```diff
@@ -5,27 +5,34 @@
   "fields": [
     {
       "field_name": "title_translated",
-      "label": "Title",
+      "label": "Organization name",
+      "extra_label": "Organization name",
+      "input_title": ["Name in Finnish", "Name in Swedish", "Name in English"],
       "preset": "fluent_core_title_translated",
       "only_default_lang_required": true,
-      "form_placeholder": "My Organization",
+      "form_placeholder": "Organization name",
       "form_languages": ["fi", "sv", "en"],
+      "show_url": true,
       "display_snippet": null
     },
     {
       "field_name": "name",
       "label": "URL",
       "validators": "not_empty unicode name_validator group_name_validator",
-      "form_snippet": "slug.html",
+      "form_snippet": null,
       "form_placeholder": "my-theme"
     },
     {
       "field_name": "description_translated",
-      "label": "Description",
-      "preset": "fluent_core_markdown_translated",
+      "label": "Organisation description",
+      "extra_label": "A general, concise, and easy-to-understand description of the organization",
+      "input_title": ["Description in Finnish", "Description in Swedish", "Description in English"],
       "validators": "fluent_text mark_as_modified_in_catalog_if_changed",
-      "form_placeholder": "A little information about my organization...",
-      "form_languages": ["fi", "sv", "en"]
+      "required": true,
+      "only_default_lang_required": true,
```

Comment:
only_default_lang_required attribute only affects if the related validator is used. required attribute still makes everything mandatory. However as I said in flowdock, we need to leave this completely optional as descriptions do not come with ListMembers.

Reply:
Done